device/bgp/bgpadvanced.yaml

```diff
@@ -36,4 +37,3 @@ components:
             If null or an empty string then authentication will not be enabled.
           type: string
           default: ''
```

Comment:
default can be null instead of empty string?

Reply:
should be null, implying usage of plain text (and hence description should be changed). empty string should be treated as an invalid key.

Reply:
ok